marketing_security_group/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============================
+Marketing security extra rules
+==============================
+
+This module add a this security features:
+
+* ACL for allowing marketing user group to remove mass.mailing objects
+* Security rule to allows marketing user to delete only his not sent mail.mass_mailing objects
+* Security rule to allows marketing manager to delete all not sent mail.mass_mailing objects
+
+For sent mail.mass_mailing objects, only an user in Administration / Settings group
+could remove them, like Odoo standard defines
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/205/8.0
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/social/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/social/issues/new?body=module:%20marketing_security_group%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+License
```

Comment:
This section is not necessary. 
Please stick to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst